src/EnergyPlus/HeatBalanceIntRadExchange.cc

```diff
@@ -1676,22 +1696,39 @@ namespace HeatBalanceIntRadExchange {
                 }
                 Real64 const sum_FixedF(sum(FixedF));
                 FinalCheckValue = FixedCheckValue = CheckConvergeTolerance = std::abs(sum_FixedF - N);
+                RowSum = sum_FixedF;
                 if (CheckConvergeTolerance > 0.005) {
+                    if (CheckConvergeTolerance > 0.1) {
+                        ShowSevereError(state,
+                                        "FixViewFactors: View factors convergence has failed "
+                                        "and will lead to heat balance errors in zone=\"" +
+                                            enclName + "\".");
+                        severeErrorPresent = true;
+                    }
                     ShowWarningError(state,
-                                     "FixViewFactors: View factors not complete. Check for bad surface descriptions or unenclosed zone=\"" +
+                                     "FixViewFactors: View factors not complete. Check "
+                                     "for bad surface descriptions or unenclosed zone=\"" +
                                          enclName + "\".");
                     ShowContinueError(state,
-                                      format("Enforced reciprocity has tolerance (ideal is 0)=[{:.6R}], Row Sum (ideal is {})=[{:.2R}].",
+                                      format("Enforced reciprocity has tolerance (ideal is "
+                                             "0)=[{:.6R}], Row Sum (ideal is {})=[{:.2R}].",
                                              CheckConvergeTolerance,
                                              N,
                                              RowSum));
-                    ShowContinueError(state, "If zone is unusual, or tolerance is on the order of 0.001, view factors are probably OK.");
+                    ShowContinueError(state,
+                                      "If zone is unusual or tolerance is on the order of 0.001, view "
+                                      "factors might be OK but results should be checked carefully.");
+                    if (anyIntMassInZone) {
+                        ShowContinueError(state,
+                                          "For zones with internal mass like this one, this"
+                                          "can happen when the internal mass has an area that"
+                                          "is much larger than the other surfaces in the zone.");
```

Comment:
Would it be appropriate to suggest splitting the internal mass into two or more surfaces?

Reply:
I will adjust the error message to suggest splitting the thermal mass into two or more surfaces.